lib/circuitbox/circuit_breaker.rb

```diff
@@ -78,12 +78,10 @@ def run(circuitbox_exceptions: true)
     end
 
     def open?
-      if open_flag?
-        true
-      else
-        false
-      end
+      circuit_store.key?(open_storage_key)
     end
+    alias :open_flag? :open?
```

Comment:
Is there a reason for keeping ```open_flag?``` around besides not having to rename the ```open_flag?``` calls to ```open?```?

Reply:
Since this is a private method, I don't see a reason why we shouldn't. Updated, pls have a look.

Reply:
Thanks for updating. Sorry it took so long for me to get to this, my notification preferences on this repo were removed.

Reply:
Thanks for merging! Do you know anybody I can reach out to for this(https://github.com/yammer/yam/pull/54)? This one's been open for a while as well.

Reply:
@matthewshafer Any chances for a new release soon?

Reply:
@ritikesh I'll ask around.

@Sija I'll get out another pre this week. I won't have documentation and upgrade guide done this week so it won't be a final release.

Reply:
Thanks. 👍

Reply:
@matthewshafer Doesn't matter as long it's published on RubyGems :)

Reply:
@sija 2.0.0.pre2 is on rubygems, let me know if you run into any issues.

Reply:
@matthewshafer Works like a charm, thanks!